cmake/ChooseCudaCC.cmake

```diff
@@ -78,10 +78,20 @@ function(chooseCudaCC SUPPORTED_CC SUPPORTED_GENCODE_FLAGS)
   # Default setting of the CUDA CC versions to compile.
   # Shortening the lists saves a lot of compile time.
   #
-  set(CUDA_MIN_CC 20)
-  set(CUDA_MAX_CC 86)
-  if(CUDA_VERSION VERSION_GREATER_EQUAL 11.1)
+
+  # The current version last time this list was updated was CUDA 12.1.
+  if(CUDA_VERSION VERSION_GREATER_EQUAL 12)
+    set(CUDA_MIN_CC 50)
+    set(CUDA_MAX_CC 90)
+  elseif(CUDA_VERSION VERSION_GREATER_EQUAL 11.8)
+    set(CUDA_MIN_CC 35)
+    set(CUDA_MAX_CC 90)
+  elseif(CUDA_VERSION VERSION_GREATER_EQUAL 11.6)
```

Comment:
according to this https://en.wikipedia.org/wiki/CUDA#GPUs_supported it was since 11.5 (but it's Wikipedia, so I don't know how reliable it is ;-) )

Reply:
https://docs.nvidia.com/cuda/archive/11.5.0/cuda-c-programming-guide/index.html#compute-capabilities <-- I extracted all these values by looking at Table 15 here, which doesn't list an 8.7 until 11.6. But if 8.7 was a specific Jetson board or whatever I guess that could make sense.

Of course, the 87 probably has to die anyway given other discussion here...